src/js/options.js

```diff
@@ -6,7 +6,7 @@ if (typeof browser === 'undefined') {
 // Some global constants.
 const HTML = document.documentElement;
 const SETTINGS_LIST = {
-  "dark_mode":                         { defaultValue: false, eventType: 'click' },
+  "dark_mode":                         { defaultValue: 'system', values: { system: matchMedia('(prefers-color-scheme: dark)').matches + ' system', dark: 'true', light: 'false' }, eventType: 'click' },
```

Comment:
Keys can have multiple possible values by simply including a `values` object in their value. `defaultValue` will reference to one of the objects in `values` in that case.